packages/admin/admin/src/apollo/useFocusAwarePolling.ts

```diff
@@ -0,0 +1,56 @@
+import { ObservableQuery, OperationVariables, QueryHookOptions } from "@apollo/client";
+import { useEffect } from "react";
+
+type FocusAwarePollingHookOptions<TData = any, TVariables = OperationVariables> = Pick<
+    ObservableQuery<TData, TVariables>,
+    "refetch" | "startPolling" | "stopPolling"
+> &
+    Pick<QueryHookOptions<TData, TVariables>, "pollInterval" | "skip">;
+
+function useFocusAwarePolling<TData = any, TVariables = OperationVariables>({
```

Comment:
no need for those generic arguments, when you pick from ObservableQuery the default values are sufficient

Reply:
Done in 47b9cde82b5f4d6fc3e062998051ec4fffccb019.